libevmcore/Instruction.cpp

```diff
@@ -204,6 +54,8 @@ static const std::map<Instruction,  InstructionInfo> c_instructionInfo =
 	{ Instruction::ADDMOD,       { "ADDMOD",         0,     3,    1,  false,       Tier::Mid } },
 	{ Instruction::MULMOD,       { "MULMOD",         0,     3,    1,  false,       Tier::Mid } },
 	{ Instruction::SIGNEXTEND,   { "SIGNEXTEND",     0,     2,    1,  false,       Tier::Low } },
+	{ Instruction::RETURNDATASIZE,{"RETURNDATASIZE", 0,     0,    1,  false,       Tier::Low } },
+	{ Instruction::RETURNDATACOPY,{"RETURNDATACOPY", 0,     3,    0,  false,       Tier::Low } },
```

Comment:
I think I was so focused on setting the number of args right I forgot about the costs.